gensim/models/ldamodel.py

```diff
@@ -1037,6 +1072,18 @@ def load(cls, fname, *args, **kwargs):
         """
         kwargs['mmap'] = kwargs.get('mmap', None)
         result = super(LdaModel, cls).load(fname, *args, **kwargs)
+        # Load the separately stored id2word dictionary saved in json.
+        id2word_fname = utils.smart_extension(fname, '.json')
```

Comment:
please make all files for one model in a special folder, so it is easy to keep track